lib/validators/delete_rejected_email_after_days_validator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DeleteRejectedEmailAfterDaysValidator
+  MAX = 36500.freeze
```

Comment:
Is there a reason we need to freeze the integer here?

Reply:
nope, just out of habit for constants and didn't think about it